app/src/shared/components/ErrorBoundary/index.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { Component, type Node } from 'react'
+import * as Sentry from '@sentry/browser'
+
+type Props = {
+    children: Node,
+}
+
+type State = {}
+
+class ErrorBoundary extends Component<Props, State> {
+    // eslint-disable-next-line class-methods-use-this
+    componentDidCatch(error: Error, extra: Object = {}) {
+        Sentry.withScope((scope) => {
+            if (extra) {
+                scope.setExtras(extra)
+            }
+            Sentry.captureException(error)
+        })
+    }
+
+    render() {
+        const { children } = this.props
+        return children
+    }
+}
+
+export default ErrorBoundary
```

Comment:
Did you know we have a HOC for setting up error boundaries and reporting errors via [analytics.js](https://github.com/streamr-dev/streamr-platform/blob/c60f834c7df073719376e95846570fe67145d1b7/app/src/analytics.js) (i.e. not just reporting to Sentry, though I am yet to use logrocket), and handling rendering an error message: 

https://github.com/streamr-dev/streamr-platform/blob/c60f834c7df073719376e95846570fe67145d1b7/app/src/shared/utils/withErrorBoundary.jsx

Usage:

https://github.com/streamr-dev/streamr-platform/blob/c60f834c7df073719376e95846570fe67145d1b7/app/src/editor/canvas/index.jsx#L497